vaadin-crud-flow-parent/vaadin-crud-flow-integration-tests/src/test/java/com/vaadin/flow/component/crud/test/ProtectedBackendIT.java

```diff
@@ -34,7 +34,7 @@ public void tryDelete() {
         Assert.assertTrue(crud.isEditorOpen());
 
         crud.getEditorDeleteButton().click();
-        crud.$(ConfirmDialogElement.class).id("confirmDelete")
+        crud.getConfirmDeleteDialog()
```

Comment:
There are several other occurences for `"confirmDelete"` and `"confirmCancel"`, it would be great if you could update those as well.

Reply:
Done